proxy/src/main/java/com/velocitypowered/proxy/config/VelocityConfiguration.java

```diff
@@ -424,7 +424,7 @@ public static VelocityConfiguration read(Path path) throws IOException {
 
     // Create the forwarding-secret file on first-time startup if it doesn't exist
     Path defaultForwardingSecretPath = Path.of("forwarding.secret");
-    if (!path.toFile().exists() && !defaultForwardingSecretPath.toFile().exists()) {
+    if (!Files.exists(path) && !Files.exists(defaultForwardingSecretPath)) {
```

Comment:
Caution: `!Files.exists` is not equivalent to [`Files.notExists`](https://docs.oracle.com/en/java/javase/18/docs/api/java.base/java/nio/file/Files.html#notExists(java.nio.file.Path,java.nio.file.LinkOption...)) when the existence cannot be determined.

Reply:
That's why I used the !Files.exists in many projects because I read its documentation, I think that in this case it is worth to put Files.notExists (now that I think about it, maybe I misunderstood the documentation the first time I read it)